x-pack/plugin/core/src/main/resources/logs-mappings.json

```diff
@@ -1,14 +1,312 @@
+            }
+          }
+        },
+        {
+          "host_name": {
+            "path_match": "host.name",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "container_id": {
+            "path_match": "container.id",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "container_name": {
+            "path_match": "container.name",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "orchestrator_namespace": {
+            "path_match": "orchestrator.namespace",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "orchestrator_id": {
+            "path_match": "orchestrator.cluster.id",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "orchestrator_id": {
+            "path_match": "orchestrator.cluster.name",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "orchestrator_resource_id": {
+            "path_match": "orchestrator.resource.id",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "orchestrator_resource_name": {
+            "path_match": "orchestrator.resource.name",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        }
+      ],
+      "dynamic": "runtime",
+      "date_detection": false,
       "properties": {
+        "@timestamp": {
+          "type": "date",
+          "ignore_malformed": false
+        },
         "data_stream": {
           "properties": {
             "type": {
               "type": "constant_keyword",
               "value": "logs"
+            },
+            "dataset": {
+              "type": "constant_keyword"
+            },
+            "namespace": {
+              "type": "constant_keyword"
+            }
+          }
+        },
+        "log": {
+          "type": "object"
+        },
+        "error": {
+          "type": "object"
+        },
+        "event": {
+          "type": "object"
+        },
+        "trace": {
+          "type": "object"
+        },
+        "transaction": {
+          "type": "object"
+        },
+        "span": {
+          "type": "object"
+        },
+        "cloud": {
+          "type": "object"
+        },
+        "container": {
+          "type": "object"
+        },
+        "host": {
+          "type": "object"
+        },
```

Comment:
I take that we would like to use subobjects:false here but we first need to automatically collapse objects in documents (https://github.com/elastic/elasticsearch/issues/88934) ? It would be nice to not have to map these objects, which I understand are mapped only to prevent users from ingesting docs with fields that have values for host, container etc.

Reply:
> I take that we would like to use subobjects:false here but we first need to automatically collapse objects in documents (#88934) ?

Yes :)

> It would be nice to not have to map these objects, which I understand are mapped only to prevent users from ingesting docs with fields that have values for host, container etc.

Without this, the dynamic templates don't work unfortunately, it's because dynamic mapping is disabled so when ingesting `log.level`, there's no object for `log`. Probably a non-issue once we use `subobjects: false`.